lua/cfc_disconnect_interface/client/cl_interface.lua

```diff
@@ -332,18 +334,22 @@ end
 
 -- Text for internet down on body
 local function populateBodyInternetDown( body )
-    local label1 = makeLabel( body, "Please check you're still connected to the internet.", 20 )
-    local label2 = makeLabel( body, "In the meantime, ", 80 )
+    makeLabel( body, "Please check you're still connected to the internet.", 20 )
+    makeLabel( body, "In the meantime, ", 80 )
+
     return "Oops, looks like you disconnected"
 end
 
 -- Text for server down on body
 local function populateBodyServerDown( body )
+    -- TODO: do these need to be used?
+    -- local frameW, frameH = body:GetSize()
 
-    local frameW, frameH = body:GetSize()
     local restartTimeStr = "The server normally takes about " .. secondsAsTime( TIME_TO_RESTART ) .. " to restart."
-    local restartTimeLabel = makeLabel( body, restartTimeStr, 0 )
-    local curTimePreLabel = makeLabel( body, "It has been down for", 32 )
```

Comment:
This is needed for the curTimePreLabel:Think() below

Reply:
Those were marked as unused? I don't see them referenced anywhere

Reply:
Line 355 `function curTimePreLabel:Think()` It could be that it Jenkin's doesn't see that as a use, and rather just a wacky function name?